creator-node/src/snapbackSM/syncHistoryAggregator.js

```diff
@@ -52,10 +54,12 @@ class SyncHistoryAggregator {
    * @param {enum} state SYNC_STATUS.success or SYNC_STATUS.fail
    * @param {String} timeOfEvent date in structure MM-DD-YYYYTHH:MM:SS:sssZ
    */
-  static async recordSyncData ({ state, timeOfEvent, logContext }) {
+  static async recordSyncData ({ state, timeOfEvent, logContext, wallet }) {
     const logger = genericLogger.child(logContext)
 
     try {
+      if (!wallet) throw new Error(`Invalid wallet passed into syncHistoryAggregator#recordSyncData`)
```

Comment:
let's add the wallet to the err msg here for debugging purposes

Reply:
sorry this error is misleading. this only gets thrown if no wallet is passed in. ill clear up

Reply:
oh, whoops! misread this as well. no worries